src/flask_ml/flask_ml_server/models.py

```diff
@@ -130,7 +130,7 @@ class ResponseModel(BaseModel):
         description="The status of the operation, e.g., 'SUCCESS'",
         min_length=1,
     )
-    results: List[Union[TextResult, ImageResult, AudioResult, VideoResult]] = Field(
+    results: Sequence[Union[TextResult, ImageResult, AudioResult, VideoResult]] = Field(

```

Comment:
`List` results in a `mypy` error. See https://mypy.readthedocs.io/en/stable/common_issues.html#invariance-vs-covariance and https://blog.meadsteve.dev/programming/2023/09/09/typed-python-prefer-sequence-over-list/ for an explanation.